src/Wsdl/Decorator/Cache/CacheDecorator.php

```diff
@@ -16,9 +16,9 @@ class CacheDecorator extends AbstractDecorator
     protected $cache;
 
     /**
-     * @var array
+     * @var array<int, string>
```

Comment:
Jaka je vyhoda  ``array<int, string>`` oproti ``string[]``?  To prvni ve mne evokuje pocit ze je to Mapa a ten ``int`` ma nejaky vyznam, zatimco z toho druheho chapu ze se jedna o "obycejnej"  seznam.

Reply:
Výhoda asi jediná a to, že phpstan chápe úplně přesně význam toho co to je - pole s indexem int a hodnotou string. Pro mě osobně je taky lepší `string[]` když se nestarám o indexy, ale phpstan to fallbackuje na `array<int|string, string>`.
Když už jsem tam viděl array tak jsem jen doplnil generiku, klidně přepíšu ;-)